README.md

```diff
@@ -5,48 +5,19 @@ allows you to run services written against the Compute@Edge APIs on your local
 development machine, and allows you to configure testing backends for your
 service to communicate with.
 
-In the near future, Viceroy will be packaged as part of the Fastly CLI and
-integrate with other Compute@Edge workflows. But for the moment, you can use it
-by hand, as described below.
+Viceroy is normally used through the [Fastly CLI's `fastly compute serve`
+command][cli], where it it fully integrated into the Compute@Edge workflows.
+However, it is also a standalone open source tool with its own CLI and a
+Rust library that can be embedded into your own testing infrastructure.
+
+[cli]: https://developer.fastly.com/learning/compute/testing/#running-a-local-testing-server
 
 ## Installation
 
 To install Viceroy, you'll need to first
 [install Rust](https://www.rust-lang.org/tools/install) if you haven't already.
-
-### Clone the repo
-
-You'll need to get a copy of the Viceroy source code locally, by cloning the
-[Viceroy repo](https://github.com/fastly/Viceroy/) like so:
-
-```
-git clone git@github.com:fastly/Viceroy.git
-cd Viceroy
-git submodule update --recursive --init
-```
-
-### Install via Cargo
-
-Next, navigate to the `Viceroy` directory resulting from the `git clone` above,
-and run:
-
-```
-cargo install --path cli
-```
-
-You should see a message that says `Installing viceroy-cli` followed by a large
-number of compilation messages. The end result is that the `viceroy` command
-is made available on your system.
-
-### Updating Viceroy
-
-To update Viceroy, navigate to the `Viceroy` directory you cloned above, and then:
-
-```
-git pull
-git submodule update --recursive --init
-cargo install --path cli
-```
+Then run `cargo install viceroy`, which will download and build the latest
```

Comment:
First, to state the obvious: This means we'll need to publish our crates before we merge this.

Reply:
Or just after :-)

Perhaps the package rename should be broken out into a separate PR if we want to publish first.

Reply:
D'oh! Right, because the rename is in this diff. :sweat_smile: 

I'll mark this as resolved, and we can handle publishing after this lands.